### PR TITLE
[9.0][MIG] Porting mass mailing partner

### PR DESCRIPTION
**Pull request by archetipo**
moved form OCA/crm [see this PR](https://github.com/OCA/crm/pull/114)

### CODE REVIEW

File: mass_mailing_partner/hooks.py
Comment (pedrobaeza) [inline mass_mailing_partner/hooks.py]: Include `with_context(active_test=False)` for including deactivated partners (if they have e-mail)

File: mass_mailing_partner/i18n/mass_mailing_partner.pot
Comment (pedrobaeza) [inline mass_mailing_partner/i18n/mass_mailing_partner.pot]: Remove POT file

**Review by pedrobaeza — APPROVED** [review]
2 little things to do in code, but I would like if you are able to squash commits by author (Javier Iniesta in one side, OCA Transbot, and yours), because there are lot of non useful commits involved.

**Comment by archetipo** [comment]
@pedrobaeza , thanks!
done

**Comment by pedrobaeza** [comment]
Thank you very much. Let's wait for the second review to merge.

@rafaelbn, can you please review?

**Review by rafaelbn — CHANGES_REQUESTED** [review]
Hi, I cannot test it in runbot, something is broken as even choosing the database with all module none are installed.

Just go to runbot and you will see that you cannot install mass_mailing_partner

**Comment by archetipo** [comment]
@rafaelbn thanks, but runbot on check is green , the problem in your test is [old module with same name installable False](https://github.com/OCA/crm/tree/9.0/mass_mailing_partner).

**Comment by pedrobaeza** [comment]
Old module removed in OCA/crm branch and runbot build restarted

**Comment by archetipo** [comment]
@pedrobaeza thanks!

**Comment by yajo** [comment]
@rafaelbn please check again.

**Review by rafaelbn — APPROVED** [review]
Tested in runbot

**Pull request merged**
